build.xml

```diff
@@ -72,9 +72,11 @@
     <property name="dir.dist.schemata" location="${dir.dist}/schemata/"/>
     <property name="dir.lib" value="lib"/>
     <property name="dir.lib.saxon" value="${dir.lib}/saxon"/>
+    <property name="dir.lib.stylesheets" value="${dir.lib}/stylesheets"/>
     <property name="dir.lib.xerces" value="${dir.lib}/xerces"/>
     <property name="saxon.download.version" value="SaxonHE12-4"/>
     <property name="saxon.jar.file" value="saxon-he-12.4.jar"/>
+    <property name="stylesheets.download.version" value="7.56.0"/>
```

Comment:
Please shorten this variable name to `stylesheets.version` like with Xerces. That's totally sufficient. We will need to shorten it for saxon too. (If you like you can include it here)